### PR TITLE
Re-run vendoring

### PR DESCRIPTION
**Pull request by justincormack**
Latest `vndr` has changed its algorithm a bit. Also it vendors some extra things at the top level, so moved it down to where the command is as that is all it is for.

Signed-off-by: Justin Cormack <justin.cormack@docker.com>

unshorn_alpaca_grazing

### CODE REVIEW

**Comment by justincormack** [comment]
There are a few minor changes, but in fact the changes are due to it picking up `pkg/containerd` which we don't really want in the top level vendor.

@ijc is going to think about this.

**Comment by justincormack** [comment]
Ok, moved vendoring to `src/cmd/linuxkit` so it does not get confused by packages, which should have standalone vendoring.

**Comment by ijc** [comment]
Is this still effective in the root:
```
$ cat .gitattributes
vendor/* linguist-vendored
kernel/*.x linguist-language=text
```
or does the linguist-vendored one need to move down the source tree or have ** prepending or ???

Everything else LGTM, although CI seems unimpressed.

**Comment by ijc** [comment]
Oh, perhaps `docs/vendoring.md` needs an update too?

**Comment by justincormack** [comment]
I think that should all be addressed now.

File: docs/vendoring.md [hunk 1]
```diff
@@ -2,12 +2,12 @@ Vendoring
 =========
 
 The Go code in this repo depends on a number of Go libraries.
-Theses are vendored in to the `vendor` directory using [`vndr`](https://github.com/lk4d4/vndr)
+Thes are vendored in to the `src/cmd/linuxkit/vendor` directory using [`vndr`](https://github.com/lk4d4/vndr)
```
Comment (ijc) [inline docs/vendoring.md]: You corrected the typo too far ;-)

File: docs/vendoring.md [hunk 2]
```diff
@@ -47,11 +47,13 @@ go get -u github.com/LK4D4/vndr
...
 
 ```
+cd src/cmd/linuxkit
 vndr
 ```
 
 To update a single dependency:
 
 ```
+/src/cmd/linuxkit
```
Comment (ijc) [inline docs/vendoring.md]: Missing the leading `cd`

**Pull request merged**